src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    @Override
+    public String toString() {
+        return super.toString();
```

Comment:
Please create your own implementation of toString() method that will print proper information instead